reuseport/reuseport_windows.go

```diff
@@ -0,0 +1,21 @@
+package reuseport
+
+import (
+	"fmt"
+	"net"
+)
+
+// ErrNoReusePort is returned if the OS doesn't support SO_REUSEPORT.
+type ErrNoReusePort struct {
```

Comment:
Can you maybe put this error and its function into `reuseport/error.go` so we don't have duplicate code. (So you also remove these from `reuseport/reuseport.go`.

Reply:
Thats been split out into a seperate file.